gdeploylib/global_vars.py

```diff
@@ -31,7 +31,15 @@ class Global(object):
     var_file = None
     master = None
     brick_hosts = []
-    log_file = os.path.expanduser('~/.gdeploy/logs/gdeploy.log')
```

Comment:
As of now gdeploy does not work unless ~/.gdeploy/logs is created manually, it spits out Traceback. Move this out of Global, put it anywhere before logging calls open() on the log_file.